test-fixture/integration-reactive/src/main/kotlin/com/linecorp/kotlinjdsl/test/reactive/querydsl/predicate/AbstractCriteriaQueryDslPredicateIntegrationTest.kt

```diff
@@ -302,4 +305,57 @@ abstract class AbstractCriteriaQueryDslPredicateIntegrationTest<S> : CriteriaQue
         // then
         assertThat(orderItemIds).isEmpty()
     }
+
+    @Test
+    fun `exists - subquery`() = runBlocking {
+        val existFoundOrders = withFactory { queryFactory ->
+            queryFactory.listQuery<Order> {
+                val entity: EntitySpec<Order> = entity(Order::class)
+                select(entity)
+                from(entity)
+                where(
+                    exists(queryFactory.subquery<Long> {
+                        val orderGroupEntity = entity(OrderGroup::class)
+                        select(literal(1))
+                        from(orderGroupEntity)
+                        where(
+                            and(
+                                col(OrderGroup::orderGroupName).equal("orderGroup1"),
+                                col(OrderGroup::order).equal(entity)
+                            )
+                        )
+                    })
+                )
+            }
+        }
+
+        assertThat(existFoundOrders.map { it.id }).isEqualTo(listOf(order1.id))
+    }
+
+    @Test
+    fun `notExists - subquery`() = runBlocking {
+
```

Comment:
This line feed seems meaningless.

Reply:
I'll change it as below

notExists - subquery -> notExists
exists - subquery -> exists